.github/workflows/ci.yaml

```diff
@@ -1,6 +1,10 @@
 name: rosdep-ci
 
-on: [push, pull_request]
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: ['*']
```

Comment:
I think this line can be omitted. Just the `pull_request:` has worked for me in the past. Not a big deal - if you want to keep it for clarity, that's fine with me.

Reply:
Thanks for the comment. And yeah empty yaml keys always make me a bit nervous so I like that I can put something in this one.